### PR TITLE
Fe order history

### LINKED ISSUE #37: Project History Interface
When I visit my profile
I see a list of products that have been delivered to me
There are a total of 10 of the most recently delivered products in a 2x5 grid
Each product should have a picture and its slug
Next to each product picture and slug there should be a return button

- [x] Set up component
- [ ] Write query/action for products
- [ ] Add functionality for keeping/returning products

### PR DESCRIPTION
**Pull request by kip-west**
#### What's this PR do?
- Links order history FE to database
- Adds actions/reducers for this functionality
- Adds preliminary tests
     - <i>Note</i>: `action` test is not passing (yet!), and is skipped for now
#### Where should the reviewer start?
- Check out the `action` and `state` file to see the async/redux functionality; then checkout `Profile` component to see how we're loading userProducts into store & `OrderHistory` component to see how this component is connected to store.
#### How should this be manually tested?
- Run tests, open in browser & manually test.
#### Any background context you want to provide?
- After getting our endpoint for userProducts, we needed to connect this to an axios.post & get these products into Redux store
#### What are the relevant tickets?
- Closes #37 
#### Screenshots (if appropriate)
order-history-initial
#### Questions:
- How are we going to render different lists appropriately? Can we link Menu Buttons to two versions of `OrderHistory` and pass different parts of store as props to the respective component?

### CODE REVIEW

File: code/web/src/modules/user/api/actions.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { getUserProducts } from './actions'
+import axios from 'axios'
+
+jest.mock('axios')
+
+describe('actions', () => {
+    it.skip('creates GET_USER_PRODUCTS after successfully fetching products', async () => {
```
Comment (dtsehl) [inline code/web/src/modules/user/api/actions.test.js:7]: Maybe we can get this test working eventually.

**Review by dtsehl — APPROVED** [review]

**Comment by ajtran303** [comment]
Thanks for the screenshot!

**Review by ajtran303 — APPROVED** [review]

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -160,3 +158,31 @@ export function getUser(id) {
     })
   }
 }
+
+// get user's products
+export function getUserProducts(token) {
+  return dispatch => {
```
Comment (leighlars) [inline code/web/src/modules/user/api/actions.js:164]: I will need your help in my request to the server for the updated form. I'll review this code and use it for mine.

**Review by leighlars — APPROVED** [review]
Thanks, Jake. This looks good.

**Pull request merged**